requirements.txt

```diff
@@ -38,5 +45,4 @@ sphinx
 hypothesis
 python-json-logger>=0.1.8
 multi-mechanize>=1.2.0
-pytest-mock
```

Comment:
why removed? It's needed for certain tests.

Reply:
because it is there twice